setup.py

```diff
@@ -14,16 +14,71 @@
 #
 # You should have received a copy of the GNU General Public License
 # along with this program. If not, see <http://www.gnu.org/licenses/>.
-from distutils.core import setup
-
-
-setup(name='mobile_robot_simulator',
-      version='0.1',
-      description='Mobile Robot Simulator',
-      author='Daniel Garcia Vaglio',
-      author_email='degv364@gmail.com',
-      url='http://www.arcoslab.org/',
-      package_dir={'mobile_robot_simulator': ''},
-      packages=['mobile_robot_simulator', 'mobile_robot_simulator.robot_models', 'mobile_robot_simulator.tools', 'mobile_robot_simulator.vectorfield'],
-      scripts=['core.py','tools/goal_obstacle.py' ]
-     )
+
+from setuptools import setup, find_packages
+
+
+def read(filename):
+    """
+    Read a file relative to setup.py location.
+    """
+    import os
+    here = os.path.dirname(os.path.abspath(__file__))
+    with open(os.path.join(here, filename)) as fd:
+        return fd.read()
+
+
+def find_version(filename):
+    """
+    Find package version in file.
+    """
+    import re
+    content = read(filename)
+    version_match = re.search(
+        r"^__version__ = ['\"]([^'\"]*)['\"]", content, re.M)
+    if version_match:
+        return version_match.group(1)
+    raise RuntimeError('Unable to find version string.')
+
+
+def find_requirements(filename):
+    """
+    Find requirements in file.
+    """
+    import string
+    content = read(filename)
+    requirements = []
+    for line in content.splitlines():
+        line = line.strip()
+        if line and line[:1] in string.ascii_letters:
+            requirements.append(line)
+    return requirements
+
+
+setup(
+    name='mobile_robot_simulator',
+    version='0.1.1',
```

Comment:
Change to: ` version=find_version('lib/arcospyu/__init__.py')` please.

Reply:
@jcrono  I will change it to 'lib/mobile_robot_simulator/__init__.py' or do you want a single version for the entire simulator? (and use that single arcospyu file as the reference?)